en/index.md

```diff
@@ -362,6 +362,51 @@ state of each unit and service:
 ...
 ```
 
+### Placeholders in code blocks
+
+There are many times when it will be necessary for the user to supply terms or
+data to a command which cannot be known in advance, for example IP addresses or
+names. It is common to substitute such values with a placeholder, where the
+placeholder consists of delimiters that contain terms that suggest the
+necessary input. For example:
+
+```
+lxc delete <instance_name>/<snapshot_name>
+```
+
+Here, the reader is expected to substitute their values for the placeholders.
+To minimise the risk of errors, instruct the reader that such values need to be
+substituted, especially when the first placeholder is referenced.
+
+This approach is in common use and is effective for short commands, but is less
+suitable for a long block of code where the user then has to edit a multi-line
+command and search through to find the placeholders to replace. A useful
+approach in these circumstances is to adjust the code to define user-supplied
+data as environmental variables. For example:
```

Comment:
I find this to be quite wordy for something that's pretty basic. How about something like:

```suggestion
This example contains a short command, and the placeholders are clearly visible 
to the user. 

However, in longer code blocks, the placeholders become 
more difficult to manage and easier for a user to overlook. For long code 
blocks, consider defining the placeholders as environmental 
variables instead. For example:
```

Reply:
I don't think we should omit the part about instructing the reader.  Consider the audience here, who are very likely to make incorrect assumptions about what is pretty basic. If we use placeholder then we should make sure we tell people and not expect them to guess.
i will take a stab at rewording this though